src/lib/containers/home.tsx

```diff
@@ -85,7 +85,10 @@ export class Home extends React.Component<Props, State> {
   }
 
   componentDidUpdate() {
-    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
+    if (this.currentScrollOffset < 0) {
+      this.currentScrollOffset = 0
+    }
+    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: !this.currentScrollOffset })
```

Comment:
The `animated: !this.currentScrollOffset` doesn’t reveal the intent well, how about storing the result of `this.currentScrollOffset < 0` in `const didPullToRefresh` and use that for the `if` and this `animated` parameter?